apps/condo/domains/organization/schema/ResetOrganizationService.test.js

```diff
@@ -178,4 +206,36 @@ describe('ResetOrganizationService', () => {
 
     })
 
+    test('notification not send for deleted meterReportingPeriod', async () => {
+        // This test is not completely valid, because if you delete the address, the push notifications will no longer be sent.
+        const client = await makeClientWithServiceConsumer()
+        const { property, organization, serviceConsumer, resident } = client
+        const [resource] = await MeterResource.getAll(admin, {})
+
+        const [meter] = await createTestMeter(admin, organization, property, resource, {
+            accountNumber: serviceConsumer.accountNumber,
+            unitName: resident.unitName,
+            verificationDate: dayjs().add(-1, 'year').toISOString(),
+            nextVerificationDate: undefined,
+        })
+
+        await createTestMeterReportingPeriod(admin, organization, {
+            notifyStartDay: 1,
+            notifyEndDay: Number(dayjs().format('DD')),
+        })
+
+        await resetOrganizationByTestClient(admin, { organizationId: organization.id })
+
+        await sendSubmitMeterReadingsPushNotifications()
+
+        const messages = await MessageApi.getAll(keystone, {
```

Comment:
use Message from test schema and `admin` client here

Reply:
and wrap it in `waitFor` since you call async task (sendSubmitMeterReadingsPushNotifications) and wait it result